NextDom/nextdom-core
# Fix issue 1466 : Variable cleaning

fix #1466

To test : 
- postint
- check all is good
- try change theme icon in page admin/profils, tab theme
- check flag display in market plugin modale
